app/controllers/welcome_controller.rb

```diff
@@ -3,7 +3,7 @@ class WelcomeController < ApplicationController
 
   def index
     unless params[:post].nil?
-      base_info = NPCFacade.create_npc(params[:post][:score_type].downcase, params[:post][:level].to_i)
+      base_info = NpcFacade.create_npc(params[:post][:score_type].downcase, params[:post][:level].to_i)
```

Comment:
I just wanted to uncapitalize the whole of `NPC` to make it match other usages of `Npc`.